storage/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM fedora:36 as build
 
 ARG TAG=v22.05
-ARG ARCH=x86_64
+ARG ARCH=native
+ARG OPI_SPDK_CFLAGS
```

Comment:
I think you want a default value here in case this isn't called from the github workflow.

Reply:
Actually, I changed my mind.  I withdraw this comment.